OrbitClientGgp/ClientGgp.cpp

```diff
@@ -55,17 +55,21 @@ bool ClientGgp::InitClient() {
 
 // Client requests to start the capture
 bool ClientGgp::RequestStartCapture(ThreadPool* thread_pool) {
-  ErrorMessageOr<void> result = Capture::StartCapture();
-  if (result.has_error()) {
-    ERROR("Error starting capture: %s", result.error().message());
-    return false;
-  }
   int32_t pid = Capture::GTargetProcess->GetID();
   LOG("Capture pid %d", pid);
 
   // TODO: selected_functions available when UploadSymbols is included
-  std::map<uint64_t, orbit_client_protos::FunctionInfo*> selected_functions =
-      Capture::GSelectedFunctionsMap;
+  // TODO(kuebler): function selection should be handled separately by each
+  //  client.
+  absl::flat_hash_map<uint64_t, orbit_client_protos::FunctionInfo>
+      selected_functions;
```

Comment:
This is a bit confusing as `selected_functions` is not filled and the TODO is two lines above now.

Reply:
The client does not yet handle selecting functions, thus I created an empty object as a placeholder here.
However, I updated the comment to make it more clear.